app/models/cherrypick_task.rb

```diff
@@ -126,25 +132,85 @@ def add_empty_well(wells)
     private :add_empty_well
   end
 
-  def pick_new_plate(requests, template, robot, plate_purpose)
+  #
+  # Returns a list with the destination positions for the control wells distributed by
+  # using batch_id and num_plate as position generators.
+  def control_positions(batch_id, num_plate, num_free_wells, num_control_wells)
+    unique_number = batch_id
+
+    # Generation of the choice
+    positions = []
+    available_positions = (0..(num_free_wells - 1)).to_a
```

Comment:
```suggestion
    available_positions = (0...num_free_wells).to_a
```

Reply:
Didnt know that 👏

Reply:
Yay! I learned about .. and ... together and I can never remember which one's which, have to google it every time :(